lib/testers/event-generator-tester.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
      * @param {function} method - A test logic.
      * @returns {any} The returned value with the test logic.
      */
-    describe: (typeof describe === "function") ? describe : function(text, method) {
+    describe: (typeof describe === "function") ? describe : /* istanbul ignore next */ function(text, method) {
```

Comment:
Just realised that this file is not getting consumed in lib anywhere. It just a helper file to test `node-event-generator` code. @mysticatea do you know why this file is not in test folder?
Also can I move it under fixtures?